### PR TITLE
add room overview base

### LINKED ISSUE #98: Task overview view
When #95 and #97 are done, the main focus of this issue should be the integration of these components into this view. With this issue, the control elements (Screenshot #2) should also be implemented.

Depends on #95, #97 

Bildschirm­foto 2023-03-19 um 4 13 40 PM

Bildschirm­foto 2023-03-19 um 4 17 53 PM

### LINKED ISSUE #181: Remove "Status" and "Label" filter in kanban board

### PR DESCRIPTION
**Pull request by DasProffi**
closes #98
closes #181

### CODE REVIEW

File: tasks/pages/ward/[uuid].tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    beds: 'Betten',
+    roomOverview: 'Room Overview'
+  },
+  de: {
+    beds: 'Bett',
+    roomOverview: 'Raum Übersicht'
+  }
+}
+
+const BedsPage: NextPage = ({ language }: PropsWithLanguage<BedsPageTranslation>) => {
```
Comment (MaxSchaefer) [inline tasks/pages/ward/[uuid].tsx:38]:
```suggestion
const WardOverview: NextPage = ({ language }: PropsWithLanguage<BedsPageTranslation>) => {
```

File: tasks/components/layout/PatientDetails.tsx [hunk 2]
```diff
@@ -64,7 +49,7 @@ export const PatientDetail = ({
   bedsInRoom,
   patient,
   onUpdate,
-  onDischarge = noop,
+  onDischarge
```
Comment (MaxSchaefer) [inline tasks/components/layout/PatientDetails.tsx:52]: Why removing the `noop` here? Make `onUpdate` and `onDischarge` optional.

**Review by MaxSchaefer — CHANGES_REQUESTED** [review]
Very nice! 🌊🚀

https://user-images.githubusercontent.com/40192999/231274816-9e9591da-815e-4991-8b9a-77532d6fa9cb.mov

**Comment by MaxSchaefer** [comment]
@DasProffi #181

**Review by use-to — APPROVED** [review]
Great job! Keep it up!

File: tasks/pages/ward/[uuid].tsx [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+import {
+  useRoomQuery,
+  useCreateMutation,
+  useDischargeMutation,
+  useUpdateMutation
+} from '../../mutations/room_mutations'
+import { RoomOverview } from '../../components/RoomOverview'
+import { PatientDetail } from '../../components/layout/PatientDetails'
+
+type BedsPageTranslation = {
```
Comment (MaxSchaefer) [inline tasks/pages/ward/[uuid].tsx:22]: Also rename to `WardOverviewTranslation`

File: tasks/pages/ward/[uuid].tsx [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+  beds: string,
+  roomOverview: string
+}
+
+const defaultBedsPageTranslation = {
```
Comment (MaxSchaefer) [inline tasks/pages/ward/[uuid].tsx:27]: Rename to `defaultWardOverviewTranslation`

File: tasks/pages/ward/[uuid].tsx [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+  en: {
+    beds: 'Betten',
+    roomOverview: 'Room Overview'
+  },
+  de: {
+    beds: 'Bett',
+    roomOverview: 'Raum Übersicht'
```
Comment (MaxSchaefer) [inline tasks/pages/ward/[uuid].tsx:34]: `Stationsliste` bzw. `Ward list`

**Review by MaxSchaefer — CHANGES_REQUESTED** [review]

File: tasks/pages/ward/[uuid].tsx (hunk 5 above)
Comment (DasProffi) [reply tasks/pages/ward/[uuid].tsx:34]: that is confusing, under the name "Stationsliste" I would imagine a list of all Stations

**Review by MaxSchaefer — APPROVED** [review]

**Pull request merged**